samcli/hook_packages/terraform/hooks/prepare/exceptions.py

```diff
@@ -9,3 +9,25 @@ class InvalidResourceLinkingException(Exception):
     def __init__(self, message):
         msg = self.fmt.format(message=message)
         Exception.__init__(self, msg)
+
+
+class OneLambdaLayerLinkingLimitationException(Exception):
+    fmt = (
+        "Sorry, Current version fo SAM CLI could not process terraform project that contains Lambda functions that "
+        "are linked to more than one lambda layer. We could not Link {layers_list} to lambda function {function_id}"
```

Comment:
Let's not use "we". Layers x could not be linked to Lambda Function Y.

Reply:
done

Reply:
Also I don't think we should apologize ("Sorry") in error messages. I've seen some guidelines on this but can't remember where; as a customer I much rather read concise and informative errors, apologies from software feels awkward.

Reply:
Agree with @hoffa. Instead of apologizing, we can just point out that it's a limitation in current beta version, and ask customers to raise a GH issue (or we create a GH issue, link the issue here and ask customers to +1 if they need this support).